baselines/commoncontrols.md

```diff
@@ -909,13 +909,13 @@ Google Takeout services SHALL be disabled.
 
 #### GWS.COMMONCONTROLS.12.1v0.3 Instructions
 1.  Sign in to https://admin.google.com as an administrator.
-2.  Select **Account** then **Google Takeout.**
+2.  Select **Data** -\> **Data Import & Export** -\> **Google Takeout**.
 3.  Select **User access to Takeout for Google services**.
-4.  For services without an individual admin control, select **Services without an individual admin control** then **Edit.**
+4.  For services without an individual admin control, select **Services without an individual admin control** then **Edit**.
 5.  Select **Don't allow for everyone**.
```

Comment:
It just says "Don't allow" for me. Could you double check this?
```suggestion
5.  Select **Don't allow**.
```

Reply:
The "Don't allow for everyone" applies only to the "Services without an admin control" mentioned in step 4 and then in step 7 for each app it has the "Don't allow" selection

Reply:
Ah thanks I see now, you're right.